### ISSUE TITLE
Implement MRI intensity scaling to preprocessing script

### ISSUE THREAD

**Issue by MisaOgura** [MisaOgura/MRNet#1]
Detailed in the [Methods section](https://journals.plos.org/plosmedicine/article?id=10.1371/journal.pmed.1002699#sec008) of in the paper from the Standford ML group.

In this method, similar intensities will have similar tissue meaning.

Cited paper: [On standardizing the MR image intensity scale](https://www.ncbi.nlm.nih.gov/pubmed/10571928).

1. the parameters of the standardizing transformation are ‘‘learned’’ from a set of images (authors used training set exams)

2. adjustment was applied to all datasets (training, validation & hidden test) using these parameters

3. Then Pixel values were clipped to 0-255

**Comment by MisaOgura** [comment]
Implemented standardisation and normalisation authors used in their code for external validation in https://github.com/MisaOgura/MRNet/commit/72a2ae2273c9cce4ce4706736396bd02af47d9cd.

Might need to revisit in the future.
